### ISSUE TITLE
Import .props.txt materials parameters to the object?

### ISSUE THREAD

**Issue by techurious** [skarndev/umodel_tools#15]
Hey, I discovered this addon from the link here: [https://www.gildor.org/smf/index.php?topic=8549.0](https://www.gildor.org/smf/index.php?topic=8549.0)

In the description it said that it is able to extract material information from .props.txt and import .psk/.pskx. However, when using the addon, it only checks for json files. Was this feature scrapped from the project? I have exported game assets (static meshes, textures, material settings) that I'd like to import to Blender for easy texturing and rendering. Is it not possible to do with this tool?

**Comment by skarndev** [comment]
No, it is not scrapped from the project. It still parses the .props.txt. It is done [here](https://github.com/skarndev/umodel_tools/blob/main/umodel_tools/props_txt_parser.py).

**Comment by techurious** [comment]
How do you use it? Is it a script you have to run manually?

**Comment by skarndev** [comment]
No, it fetches these from the asset directory automatically on import. If you are facing the issue with materials not being generated, this might be because of the unsuitable game profile.

**Comment by techurious** [comment]
Is there a list of supported game profiles? I was trying to import Mass Effect 3 assets.

**Comment by bsimser** [comment]
The .props.txt file is exported with ueviewer but that doesn't support UE5 paks. Any idea how to produce the .props.txt file for UE5?
